src/services/alert/index.js

```diff
@@ -0,0 +1,59 @@
+const hooks = require('feathers-hooks-common');
+const cache = require('memory-cache');
+const globals = require('../../../config/globals');
+
+// add Message Provider Adapter here
+const StatusAdapter = require('./adapter/status');
+
+let messages = [];
+// durtion of cache in min
+const duration = 5;
+let success = false;
+
+/**
+ * Building Message Array
+ * @param {Adapter} ProviderAdapter
+ */
+async function addMessageProvider(ProviderAdapter) {
+	const data = await ProviderAdapter.getMessage(globals.SC_THEME);
+	messages = messages.concat(data.messages);
+	return data.success;
```

Comment:
what is success? a string? then evaluate this ==='success' so it returns a boolean

Reply:
success is a boolean and describes whether the messages/incidents could be catched.